### ISSUE TITLE
1.1 RC1 Defaults to Italian

### ISSUE THREAD

**Issue by jszaszvari** [ftiff/SplashBuddy#46]
Giving the new RC for 1.1 a go, and no matter what I set the localisation settings as, the app launches in Italian.

Checked OS Settings and Xcode project is set with "Localization native development region: en"

**Comment by ftiff** [comment]
Hi John,

Thank you for your report!
That's interesting. I cannot replicate here.

Please tell me more about your environment:
Can you attach a screenshot?
What is your macOS version? 
Can you attach the result of `defaults read /Library/Preferences/.GlobalPreferences.plist` ? I need at least AppleLanguages, AppleLocale and Country.

Are all of the strings in Italian or only a couple?

Thank you!

**Comment by jszaszvari** [comment]
@ftiff sorry for the delay in replying back to you.

I'll post a detailed report later today on how I am able to replicate the issue with more details. Just gathering some info at the moment.

**Comment by ftiff** [comment]
Thank you John!

We'd like to release it for jnuc so the soonest you send the info the better. Also, are you using the release or do you compile it from master yourself ?
Thanks

**Comment by jszaszvari** [comment]
Will have it all for you in a few hours. 

Did not know you were doing something at JNUC - Cant wait to see it! (I used to work for JAMF)

:)

I just quickly checked, It actually looks like the release version works OK, I've not been able to replicate the issue with it, But if i compile it my self I see a few minor issues - But I beleive I'm using the branch that the release was compiled from.

This might indicate a problem with my development environment. Let me just finish off testing the theory I'm working on at the moment and I'll report back in a few hours. 

I'm just going to re-pull a fresh copy and start again to be 100% sure that it's not something I've done.

Hoping that It's something that I've done so you can close this one off. Will let you know within the next few hour. Sorry!

**Comment by ftiff** [comment]
Awesome! Are you coming to jnuc?? 

I found it. Issue is on my end but not for release. I left a test localisation flag on for Italian. 

I'm on iPhone so cannot post a screenshot but it is in the build scheme in a drop down menu. 

Thank you very much. I'll fix this when I get the chance. Let me know if you find anything else!

**Comment by jszaszvari** [comment]
I wont be at JNUC this year unfortunately, I intend on going next year though. If you are there not this year, but next - I'll come and buy you a beer for this absolutely amazing and awesome project. 

Thanks so much, that was exactly what was causing the issue. I had looked everywhere except that 1 location!

I've submitted a PR with the small change that fixes this. I've made it so that it uses the System Language instead. 

https://github.com/ftiff/SplashBuddy/pull/47

I appreciate it!
